src/js/ripple/seed.js

```diff
@@ -1,21 +1,26 @@
 'use strict';
 
+/*eslint new-cap: 1*/
+
 //
 // Seed support
 //
 
 var extend = require('extend');
+var lodash = require('lodash');
```

Comment:
`var lodash = require('lodash');` is better, as it's clear that the utility library is lodash and not underscore (which started this convention as far as I'm aware)